src/app/models/constraint-models/combination-constraint.ts

```diff
@@ -101,7 +101,7 @@ export class CombinationConstraint extends Constraint {
 
   clone(): CombinationConstraint {
     const clone = new CombinationConstraint(
-      this.children.map(child => child.clone()),
+      this.children.filter(child => child).map(child => child.clone()),
```

Comment:
why is this filter added?

Reply:
Sometimes I was getting `null` children. I don't understand why, actually. I will try what happens if I remove it.